code/modules/modular_computers/computers/modular_computer/variables.dm

```diff
@@ -62,6 +63,7 @@
 	var/obj/item/computer_hardware/hard_drive/portable/portable_drive		// Portable data storage
 	var/obj/item/computer_hardware/ai_slot/ai_slot							// AI slot, an intellicard housing that allows modifications of AIs.
 	var/obj/item/computer_hardware/tesla_link/tesla_link					// Tesla Link, Allows remote charging from nearest APC.
+	var/obj/item/device/paicard/personal_ai									// Personal AI, can control the device via a verb when installed
```

Comment:
ditto above

Reply:
latest commit should handle this, since it gets shat out on destroy